File: Easy/LCE - 1309.js
/**
 * @param {string} s
 * @return {string}
 */
var freqAlphabets = function(s) {
    
    let ans = '', num = 0
    let alpha = ['', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
    
    for(let i=0; i<s.length; i++) {
        if(s[i+2] === '#') {
            num = Math.floor(s[i]+s[i+1])
            i+=2
        } else {
            num = Math.floor(s[i])
        }
        
        ans += alpha[num]
    }
    
    return ans
};

File: Easy/Python - Solutions/LCE_1309.py
class Solution:
    def freqAlphabets(self, s: str) -> str:
        curr = 0
        ans = ""

        while curr < len(s):
            if curr+2 < len(s) and s[curr+2] == "#":
                ans += chr(int(s[curr:curr+2]) + 96)
                curr += 3
            else:
                ans += chr(int(s[curr]) + 96)
                curr += 1

        return ans
